www/include/core/login/processLogin.php

```diff
@@ -99,29 +99,36 @@
         $centreon = new Centreon($centreonAuth->userInfos);
         $_SESSION["centreon"] = $centreon;
 
-        $query = "INSERT INTO `session` (`session_id` , `user_id` , `current_page` , `last_reload`, `ip_address`) " .
-            "VALUES (?, ?, ?, ?, ?)";
-        $DBRESULT = $pearDB->prepare($query);
-        $pearDB->execute(
-            $DBRESULT,
-            array(session_id(), $centreon->user->user_id, '1', time(), $_SERVER["REMOTE_ADDR"])
-        );
-        if (!isset($_POST["submit"])) {
-            $minimize = '';
-            if (isset($_GET["min"]) && $_GET["min"] == '1') {
-                $minimize = '&min=1';
-            }
-            if (isset($_GET["p"]) && $_GET["p"] != '') {
-                header('Location: ./main.php?p=' . $_GET["p"] . $minimize);
-            } else if (isset($centreon->user->default_page) && $centreon->user->default_page != '') {
-                header('Location: ./main.php?p=' . $centreon->user->default_page . $minimize);
+        // Check for session
+        $sessionId = session_id();
+        if (!preg_match('/^[a-zA-Z0-9,-]{26}$/', $sessionId)) {
```

Comment:
you check that the PHPSESSID is exactly 26 characters long.
but I can't find were you check that the PHPSESSID is the same as the one set by the session ? (eg: not modified by a user to use someone else identity)